test/unittest/StatisticsBackend/InitMonitorFactoryFailsTests.cpp

```diff
@@ -147,12 +147,14 @@ class init_monitor_factory_fails_tests : public ::testing::Test
 
         // The factories will by default return mocked instances
         domain_participant_factory_ = DomainParticipantFactory::get_instance();
+        // Mock method create_participant to return this participant
+        domain_participant_factory_->domain_participant = &domain_participant_;
 
-        ON_CALL(*domain_participant_factory_, get_default_participant_qos())
-                .WillByDefault(ReturnRef(domain_participant_qos_));
+        // ON_CALL(*domain_participant_factory_, get_default_participant_qos())
```

Comment:
There are several commented lines on this file. I guess we can remove them? Could you clarify?

Reply:
Commented lines could be removed. This is because DomainParticipantFactory is no longer a Mock but a Fake class. I leave a comment in code to remember it